src/components/Mechanism/shared_properties.jsx

```diff
@@ -23,6 +23,12 @@ const options = [
     "data-type": "string",
     value: "CONSTANT",
   },
+  {
+    displayName: "diffusion coefficient",
+    name: "diffusion coeff [m^2 s-1]",
+    "data-type": "number",
+    value: 0,
+  },
```

Comment:
I really don't know about `jsx` format but this looks like key-value pairs. Why is data-type alone in the double quotation marks? Also formatting looks quite inconsistent when we look at `displayName` and `data-type`. Would it be possible to make it consistent?

Reply:
These are just json objects. You can't have a `-` in the key like `data-type`, but it can be quoted. And yes, we can update this, i'll do that now

Reply:
done